src/test/java/com/wikia/webdriver/elements/fandom/FandomPage.java

```diff
@@ -5,4 +5,6 @@
 public abstract class FandomPage<T> extends BasePageObject {
 
   public abstract T open();
+
+  public static final String SITE_URL = "http://qa.fandom.wikia.com/";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1213)